server/models.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require('mongoose');
+
+const UserSchema = mongoose.Schema(
```

Comment:
This is a great start, but a few other things to consider storing:
- First and Last Name
- Profile picture (this would be a string to a URL)
- Bio?